src/styles/partials/components/_emphasized-image.scss

```diff
@@ -0,0 +1,41 @@
+.dg_card-image {
+  background-color: $white;
+  background-position: right;
+  background-repeat: no-repeat;
+  display: block;
+  position: relative;
+  max-width: 660px;
+  min-height: 285px;
+  border-top: $border-width-large solid $red;
+}
+
+.dg_card-content {
+  display: block;
+  position: absolute;
+  background-color: $white;
+  bottom: -1px;
+  max-width: 410px;
+  width: 100%;
+  font-weight: $font-bold;
+  text-transform: uppercase;
+  font-family: "Montserrat", sans-serif;
```

Comment:
you should be able to use a variable here.

Reply:
opps missed that one

Reply:
Done